vendor/github.com/dgryski/go-farm/.travis.yml

```
@@ -0,0 +1,39 @@
+language: go
+
+sudo: false
+
+branches:
+  except:
+    - release
+
+branches:
```

YAMLlint (1.35.1)

[error] duplication of key "branches" in mapping

(key-duplicates)